.github/workflows/pull.yml

```diff
@@ -90,6 +90,7 @@ jobs:
       matrix:
         dtype: [fp32]
         build-tool: [buck2, cmake]
+        mode: [portable, xnnpack]
```

Comment:
can we select quant mode rather than portable/xnnpack? Right now this is choosing xnnpack but its actually quantizing the model. Instead I think we should have quant mode be what's selected, and XNNPACK will be the quantized backend which runs it.

Reply:
Let's update in follow on, as we also want the pt2e quant path.